travel_platform/api/skyscanner/skyscanner_data.py

```diff
@@ -0,0 +1,37 @@
+from skyscanner.skyscanner import FlightsCache
+from conf import config
+
+api_key = config.get('api_skyscanner', 'api_key')
+flights_cache_service = FlightsCache(api_key)
+
+data = {"market": "UK", "currency": "GBP", "locale": "en-GB",
+        "originplace": "SIN-sky", "destinationplace": "KUL-sky",
+        "outbounddate": "2017-07-15", "inbounddate": "2017-07-18"}
+
+
+class SkyscannerData(object):
+
+    @staticmethod
+    def cheapest_quotes(params):
+        result = flights_cache_service.get_cheapest_quotes(**params).parsed
+        return result
+
+    @staticmethod
+    def cheapest_price_by_route(params):
+        result = flights_cache_service.get_cheapest_price_by_route(**params).parsed
+        return result
+
+    @staticmethod
+    def cheapest_price_by_date(params):
+        result = flights_cache_service.get_cheapest_price_by_date(**params).parsed
+        return result
+
+    @staticmethod
+    def grid_prices_by_date(params):
+        result = flights_cache_service.get_grid_prices_by_date(**params).parsed
+        return result
+
+
+D = SkyscannerData()
```

Comment:
It looks like a testing data.
As you've added tests, please remove this part of code.